study.md

```diff
@@ -87,7 +87,19 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
+def Movie
+
+  def initialize(id, title, rating)
```

Comment:
This is unnecessary and we never want to specify `id`s since a database table with a primary key will do it for us.